diff --git composer.js composer.js
index f2bd5aa..d9ded58 100644
--- composer.js
+++ composer.js
@@ -653,7 +653,11 @@ function composeJoint(params){
 			var naked_payload_commission = objectLength.getTotalPayloadSize(objUnit); // without input coins
 
 			if (bGenesis){
-				objPaymentMessage.payload.inputs = [{type: "issue", serial_number: 1, amount: constants.TOTAL_WHITEBYTES, address: arrWitnesses[0]}];
+				var issueInput = {type: "issue", serial_number: 1, amount: constants.TOTAL_WHITEBYTES};
+				if (objUnit.authors.length > 1) {
+					issueInput.address = arrWitnesses[0];
+				}
+				objPaymentMessage.payload.inputs = [issueInput];
 				objUnit.payload_commission = objectLength.getTotalPayloadSize(objUnit);
 				total_input = constants.TOTAL_WHITEBYTES;
 				return cb();
