diff --git network.js network.js
index b52c038..41d62e1 100644
--- network.js
+++ network.js
@@ -2303,7 +2303,10 @@ function handleRequest(ws, tag, command, params){
 			// if i'm always online and i'm my own hub
 			if (bToMe){
 				sendResponse(ws, tag, "accepted");
-				eventBus.emit("message_from_hub", ws, 'hub/message', objDeviceMessage);
+				eventBus.emit("message_from_hub", ws, 'hub/message', {
+					message_hash: objectHash.getBase64Hash(objDeviceMessage),
+					message: objDeviceMessage
+				});
 				return;
 			}
 			
